### DISCUSSION TITLE
Sketchybar blocks desktop icon...

### DISCUSSION THREAD

**Discussion by jinquan-shi** [FelixKratz/SketchyBar#530] [Q&A] [answered]
Hi,

I just found that when I connect to the external display, Sketchybar will block the icons on the external display. But on the built-in display of Mac, it's okay. Has anyone encountered the same problem?

<img width="1920" alt="Snipaste_2024-04-02_13-39-18" src="https://github.com/FelixKratz/SketchyBar/assets/29059262/fa8b50a3-b992-459e-9f56-4749ece310e3">

**Comment by tomassliz** [comment, marked as answer]
I'm experiencing the same issue. It looks like macOS places the stacked folders in the upper right corner, and there is no way to change it. It happens only when you have the "Use Stacks" option selected:
![Screenshot 2024-11-27 at 14 42 57](https://github.com/user-attachments/assets/e1a62eca-eaa3-4ee6-aed6-cf70f880fd76)
